pkgs/by-name/xf/xfox/package.nix

```
@@ -0,0 +1,32 @@
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "xfox";
+  version = "1.0.0";
+  pyproject = true;
+  src = fetchFromGitHub {
+    owner = "pxlman";
+    repo = "xfox";
+    tag = "v${version}";
+    sha256 = "sha256-gbcJ02ZaPqiCo0VHnVOYzrWR00dQXgKavFTgP+6rS34=";
```

```suggestion
    hash = "sha256-gbcJ02ZaPqiCo0VHnVOYzrWR00dQXgKavFTgP+6rS34=";
```
non-specific name is preferred